src/libs/httpclient/creporternwsessionmgr.cpp

```diff
@@ -48,8 +48,14 @@ class CReporterNwSessionMgrPrivate
     //! @return Manager of the network configurations provided by the system.
     static QNetworkConfigurationManager &networkManager();
 
+    //! @return USB mode manager
+    static QUsbModed &usbModed();
+
     //! @return True if default connection is in disconnected state.
     static bool connectionIsActive();
```

Comment:
Wait, what, connection is active when connection is disconnected? Code seems to check activity so comment only wrong?